ide/che-core-ide-app/src/main/java/org/eclipse/che/ide/navigation/NavigateToFilePresenter.java

```diff
@@ -39,12 +39,16 @@
 @Singleton
 public class NavigateToFilePresenter implements NavigateToFileView.ActionDelegate {
 
+  private static final int TYPING_PERIOD_DELAY_MS = 400;
```

Comment:
To me `TYPING_REQUEST_DELAY_MS` is clearer.

Reply:
It is not a request

Reply:
query?

Reply:
It is not a query. The idea is to identify that typing is not finished yet.

Reply:
I see